next/app/[[...slug]]/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { loadDocumentPropsCached, NodeRenderer } from '@networkteam/zebra/server';
+import { notFound } from 'next/navigation';
+
+const Page = async ({ params: { slug } }: { params: { slug: string[] } }) => {
+  const routePath = slug && Array.isArray(slug) ? slug.join('/') : '/';
+
+  const neosData = await loadDocumentPropsCached(routePath);
+
+  if (!neosData) {
+    return notFound();
+  }
+
+  // TODO Handle shortcut!
```

Comment:
This could be handled within the DocumentShortcut component. For example:

```
  if (!inBackend) {
    return redirect(node?.properties.targetUri || '/');
  }
```

Assuming the `targetUri` is correctly determined by fusion.